src/app.js

```diff
@@ -1,21 +1,35 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const PORT = process.env.port || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const filePath = normalizedURL.pathname;
+  const fileName = filePath.slice(6) || 'index.html';
+
+  if (!filePath.startsWith('/file')) {
+    res.statusCode = 400;
+    res.end('The pathname has to start with "/file/"');
+
+    return;
+  }
+
+  fs.readFile(`../public/${fileName}`, (err, data) => {
```

Comment:
It only works if we are executing from src folder, e.g:
```
cd src
node app.js
```

**BUT**, it fails to retrieve files if we are executing from project root (`node src/app.js`).

To get around this, you should provide an absolute path. You can use `__dirname` for that like this: 

get the parent folder of `__dirname` using `path.dirname(__dirname)` ([doc](https://nodejs.org/api/path.html#pathdirnamepath)) and join that with `./public/${filename}` using `path.join` like before.

Reply:
fixed)